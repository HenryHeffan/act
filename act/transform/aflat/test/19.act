defproc foo(bool ar[3]; bool ar_0, ar_1, ar_2) {
    ar = {ar_0, ar_1, ar_2};
}

foo x;
