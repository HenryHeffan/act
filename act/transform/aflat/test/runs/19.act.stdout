= "x.ar[0]" "x.ar_0"
= "x.ar[1]" "x.ar_1"
= "x.ar[2]" "x.ar_2"
